docs/resources/identity_provider.md

```diff
@@ -38,6 +38,18 @@ Required:
 - `password` (String, Sensitive) User password.
 - `username` (String) User name.
 
+<a id="nestedatt--gitlab"></a>
```

Comment:
This is an autogenerated file... did you use the "make generate" command?

Reply:
Thanks for feedback. I didn't, I have now ran "make generate" to generate the file this time.

Reply:
Also added subsystem test for gitlab IDP